packages/lwt/lwt.2.5.0/opam

```diff
@@ -0,0 +1,43 @@
+opam-version: "1.2"
+maintainer: "Jérôme Vouillon <jerome.vouillon@pps.univ-paris-diderot.fr>"
+authors: [
+  "Jérôme Vouillon"
+  "Jérémie Dimino"
+]
+homepage: "http://github.com/ocsigen/lwt"
+dev-repo: "git://github.com/ocsigen/lwt"
+bug-reports: "http://github.com/ocsigen/lwt/issues"
+license: "LGPL with OpenSSL linking exception"
+build: [
+  ["./configure"
+    "--prefix" prefix
+    "--%{conf-libev:enable}%-libev"
+    "--%{camlp4:enable}%-camlp4"
+    "--%{react:enable}%-react"
+    "--%{ssl:enable}%-ssl"
+    "--%{base-unix:enable}%-unix"
+    "--%{base-threads:enable}%-preemptive"
+    "--%{lablgtk:enable}%-glib"
+    "--%{ppx_tools:enable}%-ppx"]
+  [make "build"]
+  [make "install"]
```

Comment:
Could you move this to `install`, please? It will improve parallelizability of builds.